libs/elements/layout/src/lib/components/elewa-hero/elewa-hero.component.html

```diff
@@ -0,0 +1,13 @@
+<div id="elewa-hero">
+  <img class="backgroundImage" [src]="backgroundImg" alt="top banner hero image" />
+
+  <div class="container">
+    <div>
+      <div class="content">
+        <h2 class="subtitle">{{ subtitle }}</h2>
+        <h1 class="title">{{ title }}</h1>
+      </div>
+    </div>
+    <p class="footer">{{ footer }}</p>
+  </div>
```

Comment:
The hero contains a black overlay with about 20% opacity to make the text visible on bright backgrounds. Include this.